compiler-plugin/src/main/kotlin/arrow/meta/phases/analysis/ElementScope.kt

```diff
@@ -167,10 +138,16 @@ interface ElementScope {
   ): Property
   
   val String.property: Property
+
+  val String.`member`: Property
   
   fun propertyGetter(expression: KtExpression): PropertyAccessor
   
   fun propertySetter(expression: KtExpression): PropertyAccessor
+
+  val String.propertyAccessorGet: Scope<KtPropertyAccessor>
```

Comment:
We could change this return to our new `PropertyAccessor` scope

Reply:
yes! thanks